x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/step_detail_container.tsx

```diff
@@ -104,7 +104,7 @@ export const StepDetailContainer: React.FC<Props> = ({ checkGroup, stepIndex })
           : [],
       }}
     >
-      <EuiPanel>
+      <EuiPanel hasBorder>
```

Comment:
I'd remove the border and padding, like suggested on the Steps list.

<img width="1480" alt="Screenshot 2021-06-29 at 22 23 33" src="https://user-images.githubusercontent.com/4104278/123864138-ae3cb500-d92a-11eb-904b-8ebd5af098d7.png">